### PR TITLE
600 Refactor supported versions

### PR DESCRIPTION
**Pull request by abbyad**
For #600

### CODE REVIEW

**Comment by abbyad** [comment]
@mrjones-plip fresh perspective would be useful on this PR. 

Some of my questions:
- Does the dependency and requirements language resonate here? 
- Is the dependency table still accurate and relevant? 
- Should the device requirements be moved out elsewhere, perhaps as requirements under CHT Apps?

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
lgtm! one, one char fix needed though

**Comment by mrjones-plip** [comment]
oop!  I missed a few questions there - sorry! tl;dr - no changes needed I think. Musings below.

> Does the dependency and requirements language resonate here?

Yes, good a location as any.  BTW - [nice addition](https://github.com/medic/cht-docs/pull/606/files#diff-8c0de645660ebb7683c2ed7f4684fed5c20291fe185749b38946d3e010a4eadfR53) to the hosting requirements!

> Is the dependency table still accurate and relevant?

yes to this too - it even is future proof with `4.0` in there :rocket: . The only minor tweak, now that I'm looking at it closely, is that all other tables/lists on this page are descending sorted and this is ascending sorted.  Maybe worth a re-order?  Very minor.

> Should the device requirements be moved out elsewhere, perhaps as requirements under CHT Apps?

Now that you have me looking at it - I could see an argument to move the "Requirements" section (dependencies & client devices) as a peer of "Releases" maybe? But it's such a small amount of content - it's fine where it is.

I do with the "Release Notes" could be in columns though to save on vertical scrolling. No love though [without some new templates](https://timmastny.com/blog/intro-hugo-blogdown-chef/) (ironically, no two column [on their home page](https://timmastny.com/)?).

File: content/en/core/releases/_index.md [hunk 1]
```diff
@@ -1,12 +1,71 @@
 ---
-title: "Release notes and information"
+title: "Core Framework Releases"
 linkTitle: "Releases"
 weight: 2
 description: >
+  Versions currently supported, dependencies, and release notes for the CHT Core Framework
 no_list: true
+aliases:
+    -    core/overview/supported-software
```
[the remark this answers is not in the document]
Comment (abbyad) [reply content/en/core/releases/_index.md:9]: Good eye on the slash! The leading spacing was off in my commit, and will match the spacing between the dash and path to that of the other uses of `aliases`.

**Comment by abbyad** [comment]
Thanks for the feedback @mrjones-plip! I have gone ahead and reordered the dependencies as you suggested.

**Review by mrjones-plip — APPROVED** [review]
lgtm!

**Pull request merged**